frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue

```diff
@@ -6,12 +6,14 @@ import { useNetworkStore } from '~/stores/useNetworkStore'
 const { currentNetwork, availableNetworks, isNetworkDisabled } = useNetworkStore()
 const { t: $t } = useI18n()
 
+const emit = defineEmits<{(e: 'next' | 'back'): void }>()
 const network = defineModel<ChainIDs>('network', { required: true })
-const selection = usePrimitiveRefBridge<ChainIDs, `${ChainIDs}`|''>(network)
+const selection = usePrimitiveRefBridge<ChainIDs, `${ChainIDs}`|''>(network, o => `${o || ''}`, c => Number(c || 0))
```

Comment:
what is `o =>` for?

Reply:
In the vocabulary of our bridge system, we have the "original ref" and the "created ref". They correspond to the ref that you give to the bridge and the ref that it creates.

Maybe I should rename `o` and `c` to something like `net` and `sel` to match the names of the v-models in the file?

Reply:
I renamed. Tell me if the names are still not clear enough.

Reply:
imo best would be to write it out completely

Reply:
I disagree, they would have the same names as the global varianbles although they are local. Confusing practice.

Reply:
in this case I would suggest `_network` and `_selection`?

Reply:
I think we should reserve starting with `_` only for variable that are not used.  (Example: `..forEach((_value,index)=>console.log('i am only interested in the', index))`

`net` and `sel`should be sufficient here to get the meaning out of the context in that line.